dbms/tests/performance/arithmetic.xml

```diff
@@ -62,7 +62,16 @@
         ;
     </create_query>
 
-    <query>SELECT count() FROM nums WHERE NOT ignore({op}({arg}, {arg}))</query>
+    <query>SELECT count() FROM nums WHERE NOT ignore(
+        identity({op}({arg}, {arg})),
```

Comment:
We have common subexpression elimination.

Reply:
Do we have any way to prevent it? It's surprisingly aggressive:
```
postgres=# select random(), random();
       random       |       random       
--------------------+--------------------
 0.7039404767443997 | 0.3972125559709845


:) select rand(), rand()
┌───rand()─┬───rand()─┐
│ 73369679 │ 73369679 │
└──────────┴──────────┘
```

Reply:
Not that we should imitate postgres here:
```
postgres=# select random(), random() order by random();
       random       |       random       
--------------------+--------------------
 0.5167501750407659 | 0.5167501750407659
```

Reply:
Every mature SQL engine has CSE. But both MySQL and Postgres also use weird tricks to cancel it in some cases.

Reply:
If you want to calculate the same expression multiple times:
- why don't just increase the size of the table?
- or: write +1, +2, ... for the second arg.

Reply:
The table is relatively wide, and it needs to be really big because the operations are lightweight, so I'm running out of memory. I'll try the +1 way. 

Postgres has a notion of "volatile" functions that must be executed "as written", and uses it to prevent CSE, evaluation to constant, etc. We will probably need something like that, eventually.